config/v1/types_cluster_operator.go

```diff
@@ -188,6 +188,14 @@ const (
 	// allow updates when this condition is not False, including when it is
 	// missing, True, or Unknown.
 	OperatorUpgradeable ClusterStatusConditionType = "Upgradeable"
+
+	// EvaluationConditionsDetected is used to indicate the result of the detection
+	// logic that was added to a component to evaluate the introduction of an
+	// invasive change that could potentially result in highly visible alerts,
+	// breakages or upgrade failures on user  clusters. You can concatenate
```

Comment:
nit: two spaces in `user  clusters`.

Reply:
also "on user clusters" is probably not necessary, because all of these ClusterOperator types only ever exist in OpenShift clusters.

Reply:
Removed. Please lgtm @wking.